camel/societies/role_playing.py

```diff
@@ -243,10 +266,24 @@ def get_sys_message_info(
         sys_msg_meta_dicts = [dict(task=self.task_prompt) for _ in range(2)]
         if (extend_sys_msg_meta_dicts is None and self.task_type
                 in [TaskType.AI_SOCIETY, TaskType.MISALIGNMENT]):
-            extend_sys_msg_meta_dicts = [
-                dict(assistant_role=assistant_role_name,
-                     user_role=user_role_name) for _ in range(2)
-            ]
+            if (assistant_description is not None
```

Comment:
This logic is getting out of control. Can we make a bird eye view on this and understand do we really mean that much complexity that results in such a bulky code?